src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/policies/PolicyV2Dot1.kt

```diff
@@ -525,14 +525,52 @@ val POLICY_V2_1 = LicencePolicy(
         code = "42f71b40-84cd-446d-8647-f00bbb6c079c",
         inputs = listOf(
           Input(
-            label = "Enter name or type of premises",
-            name = "nameOfPremises",
-            type = TEXT,
-          ),
-          Input(
-            label = "Enter the address of the premises",
-            name = "premisesAddress",
-            type = ADDRESS,
+            label = "Choose what information to enter",
+            name = "nameTypeAndOrAddress",
+            options = listOf(
+              Option(
+                conditional = Conditional(
+                  inputs = listOf(
+                    ConditionalInput(
+                      label = "Name or type of premises",
+                      name = "nameOfPremises",
+                      type = TEXT,
+                    ),
+                  ),
+                ),
+                value = "Just a name or type of premises",
+              ),
+              Option(
+                value = "Just an address",
+                conditional = Conditional(
+                  inputs = listOf(
+                    ConditionalInput(
+                      label = "Enter the address of the premises",
+                      name = "premisesAddress",
+                      type = ADDRESS,
+                    ),
+                  ),
+                ),
+              ),
+              Option(
+                value = "A name or type of premises and an address",
+                conditional = Conditional(
+                  inputs = listOf(
+                    ConditionalInput(
+                      label = "Name or type of premises",
+                      name = "nameOfPremises",
```

Comment:
If you haven't already, you may need to test what happens if you enter a value either here or in `Name or type of premises` then go back via the `Change` link and switch to the other option.
If I recall, a similar change that I made caused problems when that behaviour happened, as both inputs shared the same `name`.
It may not be an issue for you here though 🤞

Reply:
yeah I had to put some handling in the front end as I have two name and two address inputs on the page with the same name. So I added some js to disable the hidden inputs when the form is submitted. 

But I have now tested the back button and found another bug, due to the values being undefined, so will fix that.